Cargo.toml

```diff
@@ -440,7 +440,7 @@ revm-primitives = { version = "14.0.0", features = [
 ], default-features = false }
 
 # eth
-alloy-chains = "0.1.32"
+alloy-chains = "0.1.48"
```

Comment:
do we need to bump to "0.1.48"? I think staying on the same versions at upstream is safer.

Reply:
ok, sure

Reply:
fixed [4b92f77](https://github.com/scroll-tech/reth/pull/56/commits/4b92f778a77b8049510be02aac9ef35397b359f5)